### PR TITLE
Add basic docker-compose configuration

### PR DESCRIPTION
**Pull request by FabienM**
### What's it ?

A basic and docker-based environment for developing wallabag.

### What's included

* nginx
* PHP7 FPM 

### What's missing

* probably some php extensions (xdebug ? others ?)
* proper unix right management, keep at hand
```
sudo chown www-data -R app/cache app/logs app/bootstrap.php.cache data/db
```
* a docker-compliant firewall for `app-dev.php`

### How to test

1. make sure docker and docker-compose are installed and up-to-date
2. clone project and `composer install` it
3. `docker-compose up`
4. browse to [http://localhost:8080](http://localhost:8080/), tune rights and cache generation

### CODE REVIEW

**Comment by nicosomb** [comment]
Thank you.

:+1: for postgres, that's what I use for v2.wallabag.org and will use for production SaaS.

**Comment by j0k3r** [comment]
Yeah +1 for Postgres, maybe MySQL & SQLite for a full env.
xdebug can be a good addition too.

**Comment by nicosomb** [comment]
Do we merge like that and we'll improve later?
Or you prefer improve your PR now?

**Comment by FabienM** [comment]
Might enhance it a bit shortly.

However you can merge if you need it

**Comment by nicosomb** [comment]
Will wait for your enhancements.

**Comment by FabienM** [comment]
Updated with DBMS configuration.

* Toggle commented lines in `docker-compose.yml`.
* Configure `parameters.yml` with environment variables (example in `parameters.yml.dist`)
* Optionnally, add a `127.0.0.1 rdbms` entry to your hosts file, so you'll be able to run `wallabag:install` from your host (but in a unix right hell)

**Comment by nicosomb** [comment]
Can you add documentation in `docs/en/developer/` (this directory doesn't exist) please? (you need to rebase your branch, I just merged a PR about documentation)

**Comment by tcitworld** [comment]
Maybe more like `docs/en/administrator/` ?

**Comment by nicosomb** [comment]
> Maybe more like docs/en/administrator/ ?

It's not to deploy wallabag via docker, it's to have a dev environment via docker. Right, @FabienM ?

**Comment by tcitworld** [comment]
Indeed. My bad.

File: docker/php/env [hunk 1]
```diff
@@ -0,0 +1,6 @@
+SYMFONY__ENV__DATABASE_DRIVER="pdo_sqlite"
```
Comment (j0k3r) [inline docker/php/env]: `docker/php/env`? Shouldn't this be `docker/sqlite/env` instead?

Comment (FabienM) [inline docker/php/env]: Not sure. This file does not refer to a *sqlite* container, and could contain more `SYMFONY__` env variables or other kind of variables for *php* container.

**Comment by FabienM** [comment]
@nicosomb of course.
Remind to never work with you anymore

**Comment by FabienM** [comment]
@nicosomb `docs/en/Developer` exists but not `docs/en/developer`.
`docs/en/user` and `docs/en/User` both exist.

WTF ?

**Comment by nicosomb** [comment]
https://github.com/wallabag/wallabag/tree/v2/docs/en ?

**Comment by FabienM** [comment]
ok, that was before rebasing, my bad

**Comment by nicosomb** [comment]
Remind to never work with you anymore

**Comment by nicosomb** [comment]
Thank you!

**Pull request merged**